service-commands/scripts/init-repos.sh

```diff
@@ -37,8 +37,7 @@ cd $PROTOCOL_DIR/creator-node
 npm install
 
 cd $PROTOCOL_DIR/solana-programs/anchor/audius-data
-yarn install
-chown -R $(whoami) . # needed for some weird cargo perm err
+npm run install-dev
```

Comment:
Note that `npm run install-dev`'s script already runs yarn install and chown, so that's why I removed those 2 lines here

Reply:
yessssss